### PR TITLE
Add github readable license

### PR DESCRIPTION
**Pull request by cugu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,18 +11,3 @@ Further reading:
 * [About Kaitai Struct](http://kaitai.io/)
 * [About API implemented in this library](http://doc.kaitai.io/stream_api.html)
 * [JavaScript-specific notes](http://doc.kaitai.io/lang_javascript.html) - also includes Quick start guide
-
-## Licensing
-
-Copyright 2012-2016 Ilmari Heikkinen
-Copyright 2016-2019 Kaitai Project
```
Comment (GreyCat) [inline README.md:18]: I believe we still need to keep these two lines to trace the copyright holders.

**Comment by GreyCat** [comment]
Viewing the contents of this repo, I'm not sure if we even have any code in it.

@fudgepop01, do you have any interest in continuing this effort? I recall that https://github.com/kaitai-io/kaitai_struct_compiler/pull/165 was under review, are you still working on it?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2012-2016 Ilmari Heikkinen
+   Copyright 2016-2019 Kaitai Project
```
Comment (cugu) [inline LICENSE:190]: Here they are!

File: README.md (hunk 1 above)
Comment (cugu) [reply README.md:18]: They are in the LICENSE file. Just scroll down.

**Pull request closed without merging**